packages/wonder-blocks-button/src/components/button.stories.js

```diff
@@ -1,44 +1,289 @@
 // @flow
 import * as React from "react";
+import {StyleSheet} from "aphrodite";
+import {withDesign} from "storybook-addon-designs";
 import {action} from "@storybook/addon-actions";
 
+import {MemoryRouter, Route, Switch} from "react-router-dom";
+
+import Color from "@khanacademy/wonder-blocks-color";
 import {View} from "@khanacademy/wonder-blocks-core";
+import {icons} from "@khanacademy/wonder-blocks-icon";
 import {Strut} from "@khanacademy/wonder-blocks-layout";
-import Color from "@khanacademy/wonder-blocks-color";
-
+import Spacing from "@khanacademy/wonder-blocks-spacing";
 import type {StoryComponentType} from "@storybook/react";
 import Button from "./button.js";
 
+import ComponentInfo from "../../../../.storybook/components/component-info.js";
+import {name, version} from "../../package.json";
+
 export default {
     title: "Navigation/Button",
```

Comment:
I think this should be `title: "Navigation/Button/Basics"` or something like that so that there's better consistency between the organization of these stories and the "best practice" and "accessibility" stories.

<img width="239" alt="Screen Shot 2021-11-01 at 12 19 42 PM" src="https://user-images.githubusercontent.com/1044413/139704863-ded0954c-83e0-4793-8173-51bc0d23d545.png">

Reply:
That's a really good point! The only "downside" would be that Chromatic would think that all these stories are new (I don't think it's a problem btw, just pointing that out 😃)